### ISSUE TITLE
Unable to render map

### ISSUE THREAD

**Issue by federico-galfione** [Vheissu/aurelia-google-maps#27]
http://stackoverflow.com/questions/38426020/unable-to-render-a-google-map-using-aurelia-google-maps-plugin

Same problem here: the map doesn't render, no console errors. I've done everything written in README file without success.

**Comment by williamfelippe** [comment]
Same here.

**Comment by tavinash7** [comment]
Same here.

But I am getting this error 
**Cannot read property '__useDefault' of undefined**

**Comment by tomtomau** [comment]
Having the same issue as well. Had a dig around to try and find the culprit but couldn't find anything out of the blue so I'm assuming it's something to do with bundling?

**Comment by Vheissu** [comment]
I have a fix coming for this tonight. Sorry for the delay.

**Comment by tomtomau** [comment]
Awesome, much appreciated.

**Comment by tomtomau** [comment]
Bump - is this a simple fix I could put a PR in for?

**Comment by egeland** [comment]
I'm getting a weird issue where the installed version seems to have the `google-maps.js` file missing - from a quick look around the `dist/`, that seems to indicate that the npm/jspm version of the module uses the `amd/` folder, not the `commonjs/` folder - as the file is in `commonjs/`, but is missing in `amd/`...?

I'm about to try copying the file from the repo into the appropriate folder..

EDIT: Nope, didn't work, and looking at the `index.js` in there, seems that it has the code already, anyway.. 
So what I'm seeing is that when I bundle, by adding `aurelia-google-maps` to my `bundles.json` file, I get
![image](https://cloud.githubusercontent.com/assets/1281379/17700004/eab68ab6-63fe-11e6-9b6b-61c35e47dd97.png)
Any help would be very much appreciated. :sweat_smile:

**Comment by Vheissu** [comment]
Sorry guys it wasn't what I thought it was. Looking into this further :(

**Comment by tomtomau** [comment]
Temporary fix: #28 

Can install the patch with:

```
jspm install aurelia-google-maps=github:tomtomau/aurelia-google-maps@fix-build
```

**Comment by Vheissu** [comment]
Latest release has been pushed 1.0.14 a special thanks to the legend of the hour @tomtomau for PR'ing a fix.

**Comment by hoangcuongvn** [comment]
This error still occurs with webpack

```
Uncaught (in promise) Error: Cannot find module './aurelia-google-maps/google-maps'.
webpackContextResolve @ .*$:313
webpackContext @ .*$:308
(anonymous function) @ aurelia-loader-webpack.js:143
```

**Comment by erivan** [comment]
i have the same problem here when i try install with webpack

**Comment by spetz** [comment]
Hi,
I'm still  getting the error: `Cannot read property '__useDefault' of undefined`, my _aurelia.json_ file has the following code:

``` json
{
    "name": "aurelia-google-maps",
    "path": "../node_modules/aurelia-google-maps/dist/amd",
    "main": "index"
}
```

Is there a workaround?

**Comment by cjp666** [comment]
I changed my aurelia.json which was the same as yours to the following and no longer have the problem

```
   {
      "name": "aurelia-google-maps",
      "path": "../node_modules/aurelia-google-maps/dist/commonjs",
      "main": "index"
   }
```

**Comment by Vheissu** [comment]
I have just released a new version of the plugin 1.0.16 which addresses these issues. Plugin now solely uses TypeScript for compilation.

**Comment by ghost** [comment]
I'm seeing this issue with the 2.2.0 release when using Webpack. The map does not render, there are not any console errors, but I can see that the custom element exists and is instantiated

![image](https://user-images.githubusercontent.com/179260/35947589-944c3e46-0c36-11e8-89ac-e4a20e58d1a7.png)
